app/build.gradle

```diff
@@ -80,19 +80,25 @@ android {
     }
 }
 
+project.tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+
```

Comment:
required by the new `autodispose` version

Reply:
Resolved by accident, doesn't jvmTaget 1.8 mean android 26?..

Reply:
I have no idea tbh. We already compile Java for jvm 8, so it should be fine for Kotlin as well.